scripts/linux/install-daemon-upstart.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+
+SCRIPT_TITLE="install-daemon"
+PKG_INSTALLER="apt-get" # supports : sudo ${PKG_INSTALLER} install <pkgname> 
+
+CHK_PKGS=( "bzip2" "tar" )
```

Comment:
Wondering if we should check if `dotnet` prerequisites are installed in this script or is it out of scope?

Reply:
dotnet is no a prerequisite as RavenDB downloaded tarball contains it..

Reply:
Of course. .netcore runtime is part of the package. I meant .netcore prerequisites like `libicu` and friends.

Reply:
@aviviadi ?

Reply:
I am right now on it. answer in 10-15 minutes

Reply:
@gregolsky @ppekrol Verified on scratch install of latest raspbian stretch and RavenDB v4.1.4 - nothing was needed at all..

Reply:
Raspbian ok. However they have listed these as "required" on e.g. Ubuntu and CentOS:
https://github.com/dotnet/docs/blob/master/docs/core/linux-prerequisites.md#ubuntu
https://github.com/dotnet/docs/blob/master/docs/core/linux-prerequisites.md#centos-and-fedora

On the other hand we don't install or check C++ redist for people on Windows when doing `install-as-service.ps1` so maybe we should *not* do that here to stay consistent.
Related docs page: https://ravendb.net/docs/article-page/4.1/csharp/start/getting-started#prerequisites

Thoughts?

Reply:
Fine by me

Reply:
@gregolsky Required for SDK, not for ready to run published package.  It is like taking the safe side, but most of the packages are actually not needed in order to run RavenDB.

Reply:
well just heads up and no action required I think, but this is what I got on a fresh CentOS 7
```
FailFast:                                                                                      │
Couldn't find a valid ICU package installed on the system. Set the configuration flag System.Gl│
obalization.Invariant to true if you want to run with no globalization support.                │
                                                                                               │
   at System.Environment.FailFast(System.String)                                               │
   at System.Globalization.GlobalizationMode.GetGlobalizationInvariantMode()                   │
   at System.Globalization.GlobalizationMode..cctor()                                          │
   at System.Globalization.CultureData.CreateCultureWithInvariantData()                        │
   at System.Globalization.CultureData.get_Invariant()                                         │
   at System.Globalization.CultureInfo..cctor()                                                │
   at System.StringComparer..cctor()                                                           │
   at System.AppDomain.InitializeCompatibilityFlags()                                          │
   at System.AppDomain.Setup(System.Object)                                                    │
./run.sh: line 30: 11823 Aborted                 ./Raven.Server --browser                      │
[centos@ip-172-31-41-117 RavenDB]$
```